packages/react-components/react-spinbutton/src/components/SpinButton/SpinButton.types.ts

```diff
@@ -32,7 +32,10 @@ export type SpinButtonSlots = {
 /**
  * SpinButton Props
  */
-export type SpinButtonProps = Omit<ComponentProps<Partial<SpinButtonSlots>, 'input'>, 'onChange' | 'size'> & {
+export type SpinButtonProps = Omit<
+  ComponentProps<Partial<SpinButtonSlots>, 'input'>,
+  'onChange' | 'size' | 'value' | 'defaultValue'
```

Comment:
_nit:_ Keep these ordered

```suggestion
  'defaultValue' | 'onChange' | 'size' | 'value'
```